x/slashing/keeper/keeper.go

```diff
@@ -25,9 +25,10 @@ type Keeper struct {
 
 	// the address capable of executing a MsgUpdateParams message. Typically, this
 	// should be the x/gov module account.
-	authority string
-	Schema    collections.Schema
-	Params    collections.Item[types.Params]
+	authority            string
+	Schema               collections.Schema
+	Params               collections.Item[types.Params]
+	ValidatorSigningInfo collections.Map[sdk.ConsAddress, types.ValidatorSigningInfo]
```

Comment:
random thought but are we able to make these private? params i feel yes since the module can make a query through the router to get the same info

a bit out of scope of this pr but something to think about instead of making it all public